src/test/ui/regions/regions-close-object-into-object-4.rs

```diff
@@ -6,7 +10,14 @@ trait X { }
 impl<'a, T> X for B<'a, T> {}
 
 fn i<'a, T, U>(v: Box<dyn A<U>+'a>) -> Box<dyn X + 'static> {
-    Box::new(B(&*v)) as Box<dyn X> //~ ERROR E0759
+    Box::new(B(&*v)) as Box<dyn X> //[base]~ ERROR E0759
+    //[nll]~^ ERROR the parameter type `U` may not live long enough [E0310]
+    //[nll]~^^ ERROR the parameter type `U` may not live long enough [E0310]
+    //[nll]~^^^ ERROR the parameter type `U` may not live long enough [E0310]
+    //[nll]~^^^^ ERROR lifetime may not live long enough
+    //[nll]~^^^^^ ERROR cannot return value referencing local data `*v` [E0515]
+    //[nll]~^^^^^^ ERROR the parameter type `U` may not live long enough [E0310]
```

Comment:
Another case were emit several errors rather than one with many notes.

Reply:
Created https://github.com/rust-lang/rust/issues/96278 for this. Description is not great though, I am not sure how to phrase it better.